index.html

```diff
@@ -2,6 +2,9 @@
   <head>
     <title>Both Company</title>
     <style>
+      body {
+        background: cornsilk;
+      }
```

Comment:
There should really be a newline here

Reply:
@dmcgua01 This is absurd, I have a STRONG preference for have no line breaks between blocks, please reconsider. Have no newlines will allow us to cram more code into a single file.

Reply:
more code is better code for both company

Reply:
how do i approved the Pull Request????